spec/mercure.md

```diff
@@ -386,13 +386,8 @@ To be allowed to publish an update, the JWS presented by the publisher **MUST**
 called `mercure`, and this claim **MUST** contain a `publish` key. `mercure.publish` contains an
 array of topic selectors. See (#topic-selectors).
 
-If `mercure.publish`:
-
- *  is not defined, then the publisher **MUST NOT** be authorized to dispatch any update
-
- *  contains an empty array, the publisher **MUST NOT** be authorized to publish private updates,
-    but can publish public updates for all topics.
-
+If `mercure.publish` is not defined, then the publisher **MUST NOT** be authorized
```

Comment:
or empty?

Reply:
I don't think that it's necessary to make it explicit, as it's not the case for `subscribe` too.